nirum/rpc.py

```diff
@@ -394,7 +399,7 @@ def do_request(self, request_url, payload):
         if 200 <= response.code < 300:
             return response_text.decode('utf-8')
         else:
-            raise UnexpectedNirumResponseError(response_text)
+            raise UnexpectedNirumResponseError(response_text.decode('utf-8'))
```

Comment:
As `response_text` need to be decoded in both cases of this `if`-branch, it'd better to be decoded before this `if`-branch.

Also the name `response_text` seems misleading. Because it is not a text but a bytestring.